cloudbuild.yaml

```diff
@@ -22,7 +22,7 @@ steps:
 # Print Docker version
 #
 
-- name: "gcr.io/cloud-builders/docker"
+- name: "gcr.io/cloud-builders/docker:20.10.3"
```

Comment:
So I have to know - how did you find this?

Reply:
https://github.com/GoogleCloudPlatform/solutions-build-multi-architecture-images-tutorial/issues/1

Reply:
As a side note, I'm checking internally to see what our plans are for Docker 20.x.x - seems like this image was created in 2021, which seems like it's up for an improvement.

Reply:
As of 13 hours ago, we have Docker 20.x.x! So I think we can remove the specific tag now.

![image](https://user-images.githubusercontent.com/298370/163630337-b682c63c-0e25-4691-9be5-4d0e3a6e8627.png)